sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -3658,6 +3658,81 @@ pi_result piQueueGetInfo(pi_queue Queue, pi_queue_info ParamName,
   case PI_QUEUE_INFO_DEVICE_DEFAULT:
     die("PI_QUEUE_INFO_DEVICE_DEFAULT in piQueueGetInfo not implemented\n");
     break;
+  case PI_EXT_ONEAPI_QUEUE_INFO_STATUS: {
```

Comment:
Maybe rename to `PI_EXT_ONEAPI_QUEUE_INFO_EMPTY` since it returns true/false?
And please document returned value in pi.h.

Reply:
Sounds good, renamed and documented returned value.